File: src/main/java/web/controlevacinacao/repository/UsuarioRepository.java
package web.controlevacinacao.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import web.controlevacinacao.model.Usuario;

public interface UsuarioRepository extends JpaRepository<Usuario, Long> {
	
	Usuario findByNomeUsuarioIgnoreCase(String nomeUsuario);
	
}

File: src/main/kotlin/web/fidex/repository/UsuarioRepository.kt
package web.fidex.repository

import org.springframework.data.jpa.repository.JpaRepository
import web.fidex.model.fidex_model.Usuario

interface UsuarioRepository : JpaRepository<Usuario, Long> {
    fun findByNomeUsuarioIgnoreCase(nomeUsuario: String): Usuario?
}
